R/fixed_regex_linter.R

```diff
@@ -10,6 +10,45 @@
 #'   likely cases. It should _never_ report false positives, however; please
 #'   report false positives as an error.
 #'
+#' @examples
+#' # will produce lints
+#' code_lines <- 'gsub("\\\\.", "", x)'
+#' writeLines(code_lines)
+#' lint(
+#'   text = code_lines,
+#'   linters = fixed_regex_linter()
+#' )
+#'
+#' lint(
+#'   text = 'grepl("a[*]b", x)',
+#'   linters = fixed_regex_linter()
+#' )
+#'
+#' code_lines <- 'stringr::str_subset(x, "\\\\$")'
+#' writeLines(code_lines)
```

Comment:
WDYT about an example that doesn't use any regex special characters? e.g. `grepl("Munich", address)`